bigbite/themer
# [BBT-79] Moves plugin menu item to under the Appearance menu and renames the menu name

## Description

Fixes [BBT-79](https://b5ecom.atlassian.net/browse/BBT-79)

Moves plugin menu item to under `Appearance` in the dashboard, menu item has also been renamed to `Styles Editor`.

## Change Log

- Menu item moved to under `Appearance`
- Renamed menu item and associated strings;
  - `Theme Settings` => `Styles Editor`
  - `Theme Customiser` => `Styles Editor`
  - `theme_settings` => `styles_editor`

## Steps to test

1. Visit the Dashboard
2. Check that a new menu item `Appearance` > `Styles Editor` is available
3. Visit the above menu item and make sure the page is loading correctly
4. Check that `Settings` should no longer contain the `Theme Customiser` menu item

## Screenshots/Videos

## Checklist:



[BBT-79]: https://b5ecom.atlassian.net/browse/BBT-79?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
